docs/Using-Fleet/REST-API.md

```diff
@@ -9,7 +9,7 @@
 - [Mobile device management (MDM)](#mobile-device-management-mdm)
 - [Policies](#policies)
 - [Queries](#queries)
-- [Schedule](#schedule)
+- [Schedule (deprecated)](#schedule)
```

Comment:
@zhumo @rachaelshaw 

`POST /api/latest/fleet/global/schedule` receives a `query_id` and "schedule fields" like `interval`, `platform`, etc. Such endpoint currently (in `main` as of today) creates a new "schedule" that links to the query identified by `query_id`.

We've agreed (on a call) the following change for the new model:

`POST /api/latest/fleet/global/schedule` will now behave exactly like `PATCH /api/latest/fleet/global/schedule`, basically it will edit the "schedule fields" (`interval`, `platform`, etc.) of the query identified by the passed `query_id`.